src/pages/achievement/control/subcomponents/controlPanelTools/controlPanelUtils/TaskDeleter.tsx

```diff
@@ -0,0 +1,29 @@
+import { Button } from '@blueprintjs/core';
+import React from 'react';
+import AchievementInferencer from 'src/pages/achievement/dashboard/subcomponents/utils/AchievementInferencer';
+
+import { AchievementItem } from '../../../../../../commons/achievement/AchievementTypes';
+
+type TaskDeleterProps = {
+  editableAchievement: AchievementItem;
+  inferencer: AchievementInferencer;
+  saveChanges: any;
+};
+
+function TaskDeleter(props: TaskDeleterProps) {
+  const { editableAchievement, inferencer, saveChanges } = props;
+
+  const deleteAction = () => {
```

Comment:
Does setting it to non task also deletes it? If so, please add documentation on `setNonTask`. Else, this require better naming.

Reply:
Nope it only makes it not a task, will fix this